advanced/typescript/types.md

```diff
@@ -241,10 +241,12 @@ declarations just like they provide bundles of JavaScript as ESM.
 
 There are CDNs which host JavaScript modules that integrate well with Deno.
 
-- [Skypack.dev](https://docs.skypack.dev/skypack-cdn/code/deno) is a CDN which
-  provides type declarations (via the `X-TypeScript-Types` header) when you
-  append `?dts` as a query string to your remote module import statements. For
-  example:
+- [esm.sh](https://esm.sh) is a CDN which provides type declarations (via the
```

Comment:
esm.sh is a CDN which provides type declarations by default (via the `X-TypeScript-Types` header). You can disable it by adding `?no-dts` query:
```ts
  import React from "https://esm.sh/react?no-dts";
```